site-src/blog/2023/0801-welcome-gamma.md

```diff
@@ -0,0 +1,125 @@
+---
+description: >
+  We are excited to announce the v0.8.0 release of Gateway API, where the GAMMA
+  project has reached implementable!
+---
+
+# Gateway API: Welcome, GAMMA!
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: August 01, 2023 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We are thrilled to announce the v0.8.0 release of the Gateway API! With this
+release, the work the GAMMA initiative has been doing over the past year has
+reached [Experimental status][status], with two conformant implementations at
+present (Linkerd and Istio). We look forward to your feedback!
```

Comment:
@michaelbeaumont does Kuma meet this?

Reply:
Sure, [we're running the Mesh conformance tests in CI](https://github.com/kumahq/kuma/blob/8c09d6fe17f557b1ec75620fcf36e5c1ce44a252/test/e2e/gateway/gatewayapi/conformance_test.go#L81-L93)

Reply:
Thanks @michaelbeaumont! I'll include that as well.